### PR TITLE
Brazilian Portuguese

### PR DESCRIPTION
**Pull request by RenatohRibeiro**
New language

### CODE REVIEW

File: pt/index.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Algum especialista em Java por aí?
+      </p>
+    </blockquote>
+    <p>
+      Esta é uma má forma, por várias razões. O que a pessoa está realmente
+      <em>perguntando</em> é,
```
Comment (yangm97) [inline pt/index.html:97]: Same as above. This reads as a request so we would be better with "pedir".

File: pt/index.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Quaisquer especialistas em Java que estejam dispostos a se comprometer a investigar
+        problema, seja o que for, mesmo que não seja realmente
+        relacionado a Java ou se alguém que não sabe nada sobre Java
+        poderia realmente responder a minha pergunta?
+      </p>
+    </blockquote>
+    <p>
+      Existem muitas razões pelas quais as pessoas que TÊM conhecimento não admitir isso. Ao perguntar, você está pedindo mais do que pensa você está perguntando.
```
Comment (yangm97) [inline pt/index.html:109]: "admitir isso" what? Also, "Ao pedir para perguntar, você está pedindo mais do que imagina." would probably be

File: pt/index.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    </p>
+    <p>
+      Você está pedindo às pessoas que assumam a responsabilidade. Você está questionando as pessoas confiança em suas habilidades. Você também está murando desnecessariamente pessoas fora. Costumo responder perguntas relacionadas a idiomas ou bibliotecas que nunca foram usadas, porque as respostas são (de um modo programador) message
```
Comment (yangm97) [inline pt/index.html:112]: There's a dangling "message" at the end of the line. Also, this section sounds somewhat too literal.

File: pt/index.html [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+      }
+      blockquote > .name {
+        padding-right: 8px;
+      }
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>Não pergunte para pedir, apenas peça</h1>
```
[the remark this answers is not in the document]
Comment (rffontenelle) [reply pt/index.html:85]: "Não pergunte para perguntar, apenas pergunte" would probably be better. "Peça para perguntar" is as if users ask permission to ask, which doesn't normally happens. They ask a pre-question before asking the main one, so "ask to ask" would be more correctly translated to "pergunte para perguntar".

File: pt/index.html [hunk 5]
```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="pt">
+<head>
+  <title>Não pergunte para pedir, apenas peça</title>
+  <meta property="og:title" content="Não pergunte para pedir, apenas peça" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com" />
```
Comment (tcarreira) [inline pt/index.html:7]: should be   <meta property="og:url" content="https://dontasktoask.com/pt" />

see for reference

File: pt/index.html [hunk 6]
```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="pt">
+<head>
+  <title>Não pergunte para pedir, apenas peça</title>
```
Comment (tcarreira) [inline pt/index.html:4]: I would suggest "Não peça para perguntar, apenas pergunte"

**Pull request closed without merging**